tests/csm/rest/test_capacity_usage.py

```diff
@@ -1968,3 +2018,914 @@ def test_34723(self):
         self.log.info("##### Test ended -  %s #####", test_case_name)
+
+    # pylint: disable-msg=too-many-statements
+    @pytest.mark.skip("CORTX-30810")
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.tags('TEST-39924')
+    def test_39924(self):
+        """
+        Test degraded capacity with single node failure ( K>0 ) with IOs for 3+2+0 config with 5
+        nodes using aws
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("-------------------------Step 1 Starts-------------------------")
+        self.log.info(
+            "[Start] Fetch degraded capacity on Consul with 0 Node failure")
+        resp = self.csm_obj.get_capacity_consul()
+        cap_df = self.csm_obj.get_dataframe_failure_recovery(self.num_worker)
+        test_cfg = self.csm_conf["test_39924"]
+        total_written = resp["healthy"]
+        cap_df.loc["No failure"]["consul_healthy"] = resp["healthy"]
+        cap_df.loc["No failure"]["consul_degraded"] = resp["degraded"]
+        cap_df.loc["No failure"]["consul_critical"] = resp["critical"]
+        cap_df.loc["No failure"]["consul_damaged"] = resp["damaged"]
+        assert self.csm_obj.verify_degraded_capacity(resp, healthy=None, degraded=0,
+                                                     critical=0, damaged=0,
+                                                     err_margin=test_cfg["err_margin"])
+        self.log.info(
+            "[End] Fetch degraded capacity on Consul with 0 Node failure")
+        self.log.info(
+            "[Start] Fetch degraded capacity on HCTL with 0 Node failure")
+        resp = self.hlth_master.hctl_status_json()["bytecount"]
+        cap_df.loc["No failure"]["hctl_healthy"] = resp["healthy"]
+        cap_df.loc["No failure"]["hctl_degraded"] = resp["degraded"]
+        cap_df.loc["No failure"]["hctl_critical"] = resp["critical"]
+        cap_df.loc["No failure"]["hctl_damaged"] = resp["damaged"]
+        assert self.csm_obj.verify_degraded_capacity(
+            resp, healthy=None, degraded=0, critical=0, damaged=0,
+            err_margin=test_cfg["err_margin"],
+            total=total_written)
+        self.log.info(
+            "[End] Fetch degraded capacity on HCTL with 0 Node failure")
+        self.log.info("[Start] Fetch degraded capacity on CSM")
+        resp = self.csm_obj.get_degraded_capacity()
+        assert resp.status_code == HTTPStatus.OK, "Status code check failed."
+        resp = resp.json()["bytecount"]
+        cap_df.loc["No failure"]["csm_healthy"] = resp["healthy"]
+        cap_df.loc["No failure"]["csm_degraded"] = resp["degraded"]
+        cap_df.loc["No failure"]["csm_critical"] = resp["critical"]
+        cap_df.loc["No failure"]["csm_damaged"] = resp["damaged"]
+        healthy_count = resp["healthy"]
+        assert self.csm_obj.verify_degraded_capacity(
```

Comment:
This was incorrect in old tests. U need to use as below.

resp = self.csm_obj.verify_degraded_capacity(...)
assert resp[0], resp[1]

Reply:
okay